strongbox-data-service/src/main/java/org/carlspring/strongbox/data/service/SynchronizedCommonCrudService.java

```diff
@@ -0,0 +1,82 @@
+package org.carlspring.strongbox.data.service;
+
+import org.carlspring.strongbox.data.domain.GenericEntity;
+
+import javax.inject.Inject;
+
+import com.hazelcast.core.HazelcastInstance;
+import com.hazelcast.core.IMap;
+import com.orientechnologies.orient.core.id.ORID;
+
+public abstract class SynchronizedCommonCrudService<T extends GenericEntity>
```

Comment:
@fuss86 we already have a synchronization within Business layer, I don't think we should also have it in DAO by default on every save operation.